src/main/java/frc/robot/subsystems/conditioners/CurvesConditioner.java

```diff
@@ -3,8 +3,8 @@
 import org.usfirst.frc.team4999.utils.Utils;
 
 public class CurvesConditioner implements DriveConditioner {
-  private final double move_curve = 2;
-  private final double turn_curve = 2;
+  private final double move_curve = 1;
+  private final double turn_curve = 1;
```

Comment:
I assume this was at the driver's request.
What problem were you trying to solve?
This shouldn't affect wheelies, in theory it actually makes the robot harder to control.

Reply:
This is what I said.  I definitely saw a change in the behavior after they made this change, but it could have been as much from driver practice as from a math change.  Needs more testing.